### DISCUSSION TITLE
Why do you only allow telemetry >= 10?

### DISCUSSION THREAD

**Discussion by TheCherry** [arendst/Tasmota#22279] [Support request] [answered]
Is there any special reason for this limitation?

https://github.com/arendst/Tasmota/blob/4b06cb27b0395d62b0e52465d1cd00e65c63e835/tasmota/tasmota_support/support_command.ino#L2419

**Comment by sfromis** [comment, marked as answer]
One reason for a lower bound is about what devices can handle, not all can "survive" a lot of activity.

On a more general level, if you think that you need fast data, maybe the concept of fixed-period telemetry messages is not "the solution".

Like the typical use case of energy monitoring data for devices being switched on/off, or otherwise varying significantly in power draw. Here, Tasmota has the `PowerDelta` feature allowing you to control thresholds for when to report, which makes a lot more sense than wanting messages, say, every second.

In other words, maybe the device should be able to decide when it is "worth it" to report data.
